### DISCUSSION TITLE
BidWithComment Auction.sol upgrade

### DISCUSSION THREAD

**Discussion by EricDevito** [ourzora/nouns-protocol#119] [Ideas]
gm, working on a small upgrade that will give bidders the ability to pass comments/reasons alongside their dao token bids. Isaac recommended I pop a few words in here. I am yet to send an official PR but welcome all questions and suggestions : )

upgrade: https://github.com/EricDevito/nouns-protocol

**Comment by neokry** [comment]
Thanks for opening this discussion @EricDevito ! I see the potential with this feature but from the client side we have a lot of other stuff in the pipeline we need to work out before implementing a feat like this. would you be interested in providing the client side part of this feature as well or have another dev that would take this up?
